[stable/traefik] Traefik accesslogs filters

kenichi-shibata opened the pull request:
<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

#### What this PR does / why we need it:

It enables traefik accessLogs filters documented [here](https://docs.traefik.io/configuration/logs/)

#### Special notes for your reviewer:

#### Checklist

- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

Commit:
[stable/postgreslq] Improve service configuration (#8691)

* Fix typo

Signed-off-by: juan131 <juan@bitnami.com>

* [stable/postgreslq] Improve service configuration

Signed-off-by: juan131 <juan@bitnami.com>

* Fix lint issues

Signed-off-by: juan131 <juan@bitnami.com>

* Small corrections on README.md file

Signed-off-by: juan131 <juan@bitnami.com>

* Small corrections on README.md file - ii

Signed-off-by: juan131 <juan@bitnami.com>

Added access log filters

this allows for accessLogs filters to be enabled

Signed-off-by: Kenichi Shibata <kenichi-shibata@outlook.com>

Bump version for chart

Signed-off-by: Kenichi Shibata <kenichi-shibata@outlook.com>

Document access Logs filter

RE add accidentally deleted format

Small fixes

small fixes

Update stable/traefik/templates/configmap.yaml

Co-Authored-By: kenichi-shibata <kenichi-shibata@outlook.com>
Signed-off-by: Kenichi Shibata <kenichi-shibata@outlook.com>

mgrachev committed:
[stable/hlf-ord] fix typo (#8695)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

mgrachev committed:
[stable/hlf-peer] fix typo (#8696)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

mgrachev committed:
[stable/home-assistant] fix typo (#8702)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

hairyhenderson committed:
[prometheus-blackbox-exporter] Fixing minor syntax error (#8674)

Signed-off-by: Dave Henderson <dhenderson@gmail.com>

paskal committed:
fix typos in prometheus-operator (#8705)

* fix typos

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

* bump version

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

* fix typo

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

irlevesque committed:
[stable/datadog] Add support for mapping Kubernetes Labels and Annotations to Datadog Tags (#8643)

* Add support for mapping Kubernetes Labels and Annotations to Datadog
Tags

Signed-off-by: Ian Levesque <ian@quantopian.com>

* use YAML representation of mappings

Signed-off-by: Ian Levesque <ian@quantopian.com>

* Update default values examples

Signed-off-by: Ian Levesque <ian@quantopian.com>

mgrachev committed:
[stable/kuberos] fix typo (#8704)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

ljfranklin committed:
livenessProbe in kube-dashboard is configurable (#8621)

- Adds configurable `livenessProbe.initialDelaySeconds` and
  `livenessProbe.timeoutSeconds`

Signed-off-by: Lyle Franklin <lylejfranklin@gmail.com>

mgrachev committed:
[stable/mariadb] fix typo (#8701)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

cpanato committed:
bump minio and mc (#8570)

Signed-off-by: cpanato <ctadeu@gmail.com>

desaintmartin committed:
Guidelines: set matchLabels as being mandatory (#7692)

* Guidelines: set matchLabels as being mandatory.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* fixup! Guidelines: set matchLabels as being mandatory.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* fixup! fixup! Guidelines: set matchLabels as being mandatory.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Update guidelines: mention DaemonSets as well.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Review guidelines: be more precise + specify upgrade

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Review guidelines: fix typos, add persistence paragraph and do not repeat component part.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Review guidelines: add PVC paragraph.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Fix spelling/typos

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

* Fix incorrect typo fix

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

Commit:
7552 | Change path of health checks and add init delay (#7554)

* 7552 | Change path of health checks and add init delay

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Incrementing chart version

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Adding configuration for readinessDelay

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Updating chart version to master

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Bumping chart version.

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

vsliouniaev committed:
Update readme and examples (#8709)

Signed-off-by: Vasily Sliouniaev <vasily.sliouniaev@jet.com>

cablespaghetti committed:
Upgrade to Kafka 2.0.0 and disable confluent metric submission (#8693)

Signed-off-by: Sam Weston <weston.sam@gmail.com>

Commit:
[stable/prisma]: add service account (#8716)

Signed-off-by: Giacomo Guiulfo <giacomoguiulfo@gmail.com>

adoy committed:
[Minio] Fix domain and browser config (#8683)

Following https://github.com/minio/minio/pull/6498/files
domain and config are deprecated in config entries. Use env variable instead

Signed-off-by: Pierrick Charron <pierrick@webstart.fr>

hkaj committed:
support CRI in agent Deployment scenario (#8711)

Signed-off-by: Haissam Kaj <haissam.kaj@gmail.com>

davidkarlsen committed:
upgrade grafana (#8708)

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>

tompizmor committed:
[stable/phpmyadmin] Update phpmyadmin to version 4.8.3 (#8726)

Signed-off-by: tompizmor <tompizmor@gmail.com>

mgrachev committed:
[stable/redis] fix typo (#8699)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

mgrachev committed:
[stable/rabbitmq] fix typo (#8700)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

vsliouniaev committed:
Add missing prometheus options to resource (#8738)

- scrapeInterval
- evaluationInterval

Signed-off-by: Vasily Sliouniaev <vasily.sliouniaev@jet.com>

desaintmartin committed:
[incubator/elasticsearch] pre-stop-hook and post-start-hook drain scripts are now optional. (#8622)

We don't need to always drain a data node that is restarting.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

carrodher committed:
Use minideb as initContainer to change volumes permissions (#8737)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

desaintmartin committed:
[stable/rabbitmq] metrics readiness probe: set timeout to 5s. (#8733)

* [stable/rabbitmq] metrics readiness probe: set timeout to 5s.

1s is sometimes not enough for metrics to answer.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Bumb version.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

Commit:
[stable/magic-namespace] Adding annotations and labels to namespaces (#8324)

* Adding annotations and labels to namespaces

Signed-off-by: Garcia De La Villa, Alvaro <alvaro.garcia@lexisnexisrisk.com>

* Reverting namespace object and adding namespaceAttributes

Signed-off-by: Garcia De La Villa, Alvaro <alvaro.garcia@lexisnexisrisk.com>

* Commenting namespace value and deleting unused attributes

Signed-off-by: Garcia De La Villa, Alvaro <alvaro.garcia@lexisnexisrisk.com>

irlevesque committed:
[stable/datadog] Add support for disabling Docker socket mount requirement (#8641)

* Add support for removing Docker socket mount requirement

Signed-off-by: Ian Levesque <ian@quantopian.com>

* Use convention of lowercased 'true'

Signed-off-by: Ian Levesque <ian@quantopian.com>

spiarh committed:
[stable/prometheus] Make pod annotations configurable for PodSecurityPolicy (#8746)

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

dtomcej commented on stable/traefik/templates/configmap.yaml:
```diff
@@ -147,6 +147,18 @@ data:
+    {{- if .Values.accessLogs.filters }}
+    [accessLog.filters]
+      {{- if .Values.accessLogs.filters.statusCodes }}
+      statusCodes = [{{ range $_, $value := .Values.accessLogs.filters.statusCodes }}"{{ $value }}",{{ end }}]
```
```suggestion
      statusCodes = [{{ range $_, $value := .Values.accessLogs.filters.statusCodes }}{{ $value | quote }},{{ end }}]
```

joeauty committed:
Fix Traefik configmap for Zookeeper prefix (#8647)

* fix Traefik configmap for Zookeeper prefix

Signed-off-by: Joe Auty <joe@thinkdataworks.com>

* use quotation function across entire Traefik helm chart

Signed-off-by: Joe Auty <joe@thinkdataworks.com>

* forgot to save dashboard service change

Signed-off-by: Joe Auty <joe@thinkdataworks.com>

* another patch version bump

Signed-off-by: Joe Auty <joe@thinkdataworks.com>

desaintmartin committed:
[stable/rabbitmq] curl Probes: do no print content or progress but show error. (#8732)

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

hbagdi committed:
[stable/kong] add hbagdi and shashiranjan84 as OWNERS (#8768)

* [stable/kong] add hbagdi and shashiranjan84 as OWNERS

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

* [stable/kong] bump chart version to 0.5.1

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

yuchaoran2011 committed:
Updated apiVersions and fixed doc (#8672)

* Updated apiVersions and fixed doc

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

* Updated appVersion

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

Commit:
[stable/odoo] Adapt 'values.yaml' to new Postgres values (#8778)

* Fix typo

Signed-off-by: juan131 <juan@bitnami.com>

* [stable/odoo] Adapt 'values.yaml' to new Postgres values

Signed-off-by: juan131 <juan@bitnami.com>

Commit:
adding configmap to activate the external metrics provider (#8764)

Signed-off-by: charlyF <charly@datadoghq.com>

alexvicegrab committed:
[HLF-CA] Fix intermediate CAs (#8761)

Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

Bump ver

Signed-off-by: Kenichi Shibata <kenichi-shibata@outlook.com>

Update stable/traefik/Chart.yaml

Co-Authored-By: kenichi-shibata <kenichi-shibata@outlook.com>

Commit:
[stable/prometheus-operator] Remove unnecessary indent in serviceMonitorNamespaceSelector (#8780)

* Remove unnecessary space in serviceMonitorNamespaceSelector template

Signed-off-by: Faheem Nadeem <faheem@cliqz.com>

* Bump chart version to 0.1.7

Signed-off-by: Faheem Nadeem <faheem@cliqz.com>

alexvicegrab committed:
[HLF-Peer] Support Cryptogen as well as Fabric CA by attaching secrets (#8792)

* [HLF-Peer] Support Cryptogen as well as Fabric CA by attaching crypto material as secrets

Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

* [HLF-Peer] Remove secret

Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

alexvicegrab committed:
[HLF-Ord] Support Cryptogen as well as Fabric CA by attaching secrets (#8791)

* [HLF-Ord] Support Cryptogen as well as Fabric CA by attaching crypto material as secrets

Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

* [HLF-Ord] Remove secret

Signed-off-by: Alejandro Vicente Grabovetsky <sasha@aid.technology>

enflo committed:
Update appVersion and image tag (#8794)

Signed-off-by: Antoni Florit Homar <toniflorithomar@gmail.com>

dkozlov committed:
[stable/kong] Cassandra, "wait-for-db" initContainer failed to start: Error: [postgres error] could not retrieve server_version: connection refused (#8681)

* [stable/kong] Cassandra, "wait-for-db" initContainer failed to start: Error: [postgres error] could not retrieve server_version: connection refused

Signed-off-by: Dmitry Kozlov <dmitry.f.kozlov@gmail.com>

* [stable/kong] change appVersion to 0.14.1

Signed-off-by: Dmitry Kozlov <dmitry.f.kozlov@gmail.com>

* [stable/kong] change Chart.yaml version to 0.5.2

Signed-off-by: Dmitry Kozlov <dmitry.f.kozlov@gmail.com>

* [stable/kong] added new line in Chart.yaml

Signed-off-by: Dmitry Kozlov <dmitry.f.kozlov@gmail.com>

* [stable/kong] fix Chart.yaml

Signed-off-by: Dmitry Kozlov <dmitry.f.kozlov@gmail.com>

cilindrox committed:
[stable/kong] Add imagePullSecrets Option (#7368)

Signed-off-by: Gaston Festari <cilindrox@gmail.com>

Commit:
[stable/mongodb-replicaset] Fix regression when using passwords with spaces (#8668)

* Fix regression when using passwords with spaces

Signed-off-by: Steven Sheehy <ssheehy@firescope.com>

* Changes from code review

Signed-off-by: Steven Sheehy <ssheehy@firescope.com>

* Add terminationGracePeriodSeconds to README

Signed-off-by: Steven Sheehy <ssheehy@firescope.com>

ThorstenKunz committed:
[stable/fluent-bit] Fix ElasticSearch Output config templatate (#8166)

The most important part of this PR is the fix for the ES config which was broken because it was cut off by the Splunk config.
It also adds the tls options to the http and fluentd output plugin as well as a shared_secret option to the fluentd forwarder output plugin. Also http_user/password options are added to the http backend. All options have backwards compatible defauts.

Signed-off-by: Thorsten Kunz <thorsten.kunz@flexarc.com>

hbagdi committed:
[stable/kong] update description and links (#8796)

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

rendhalver committed:
[incubator/elastic-stack] Add fluentd-elasticsearch and elasticsearch-exporter to elastic-stack (#8616)

* Add fluentd-elasticsearch and elasticsearch-exporter to elastic-stack

Update repository for elasticseach and elasticsearch-curator

Signed-off-by: Pete Brown <pete.brown@powerhrg.com>

* Increment minor version

Signed-off-by: Pete Brown <pete.brown@powerhrg.com>

vsliouniaev committed:
Fix README.md typo and whitespace (#8612)

Signed-off-by: Vasily Sliouniaev <vasily.sliouniaev@jet.com>

guptaarvindk committed:
nexus/clusterIP-headlessService (#8565)

* support for clusterIP services with specific cluster IP or None for headless service.

Signed-off-by: Arvind Gupta <guptaarvindk@gmail.com>

* - added support for statefulsets, while keeping backward compatibility for deploymnet usage.
- fixed issues from previous PR
- bumped up minor version.
Signed-off-by: Arvind Gupta <guptaarvindk@gmail.com>

varshavaradarajan committed:
[stable/gocd] Volume mount config map even if persistence is disabled. (#8772)

* Volume mount the config map to preconfigure the server even if persistence is disabled.
* The value for .Values.server.persistence.enabled should be used for volume mounting GoCD related directories such as config, db, etc.
Preconfiguring the GoCD server is should be done even if persistence is not enabled. (#8579)

Signed-off-by: Varsha Varadarajan <varshasvaradarajan@gmail.com>

* Update changelog.

Signed-off-by: Varsha Varadarajan <varshasvaradarajan@gmail.com>

desaintmartin committed:
[stable/elasticsearch] Add myself as maintainer. (#8689)

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

Commit:
[incubator/elastic-stack] Add filebeat to elastic-stack (#8752)

* [incubator/elastic-stack] Add filebeat to elastic-stack

Signed-off-by: Dmitry Zykov <i@dmizyk.com>

* remove trailing space

Signed-off-by: Dmitry Zykov <i@dmizyk.com>

* regenerate requirements.lock

Signed-off-by: Dmitry Zykov <i@dmizyk.com>

* add new line character at the end of file Chart.yaml

Signed-off-by: Dmitry Zykov <i@dmizyk.com>

dennybaa committed:
fixed chipherSuites --> cipherSuites typo (#8806)

Signed-off-by: Denis Baryshev <dennybaa@gmail.com>

mmatur commented on stable/traefik/Chart.yaml:
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.52.3
+version: 1.52.4
```
```suggestion
version: 1.52.5
```

tompizmor committed:
[stable/magento] Expose readiness and liveness probes through values (#8832)

Signed-off-by: tompizmor <tompizmor@gmail.com>

tompizmor committed:
[stable/postgresql] Fix NOTES.txt render error when service type is LB (#8835)

* [stable/postgresql] Fix NOTES.txt render error when service type is LoadBalancer

Signed-off-by: tompizmor <tompizmor@gmail.com>

* Delete wrong line

Signed-off-by: tompizmor <tompizmor@gmail.com>

diwakar-s-maurya committed:
[stable/sonatype-nexus] create proxy service only if nexus-proxy is enabled (#8629)

Signed-off-by: Diwakar Singh Maurya <diwakar.s.maurya@gmail.com>

irozet committed:
[stable/openvpn] adding ability to keep certificates in chart values (#7916)

* Additing an option to create extra route for Kubernetes service network

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* bump chart version

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* bump minor update version

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* [stable/openvpn] fixing typo in openvpn.OVPN_K8S_SVC_SUBNET value name

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* [stable/openvpn] fix openvpn.OVPN_K8S_SVC_NETWORK and openvpn.OVPN_K8S_SVC_SUBNET names in values.yaml

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* [stable/openvpn] adding ability to keep certificates in chart values

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* passing certificates in secret rather than values

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* remove wrong values from from doc

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>

* fix typo

Signed-off-by: Irina Rozet <irina.rozet@ingrammicro.com>
